lib/config/accountsConfig.js

```diff
@@ -0,0 +1,78 @@
+var mySubmitFunc = function(error, state){
+  if (!error) {
```

Comment:
So this is just a stub that's an example given from https://github.com/meteor-useraccounts/core/blob/master/Guide.md.

If you prefer we can take it out or we could leave it in as a kind of documentation on some possible future functionality there.  Normal errors (wrong password etc.) are already handled by the package, so I don't know offhand of any error handling to put in there right now.